src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -124,6 +124,26 @@ public static CultureInfo CookieCulture(NancyContext context, GlobalizationConfi
 
             return null;
         }
+        /// <summary>
+        /// Checks to see if a default culture has been set on <see cref="GlobalizationConfiguration"/>
+        /// </summary>
+        /// <param name="context">NancyContext</param>
+        /// <param name="configuration">Culture configuration that contains allowed cultures</param>
+        /// <returns></returns>
+        public static CultureInfo ConfigurationCulture(NancyContext context, GlobalizationConfiguration configuration)

```

Comment:
Maybe call this `GlobalizationCulture` or `GlobalizationConfigurationCulture` so it is more descriptive of what it does